index.html

```diff
@@ -5,22 +5,24 @@
 	<meta charset="UTF-8">
 	<meta http-equiv="X-UA-Compatible" content="IE=edge">
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
-	<link rel="stylesheet" href="./src/style.css">
 	<title>todo list</title>
+	<link rel="stylesheet" href="./src/style.css">
 </head>
 
 <body>
 	<header>
-		<h1 class="title">todo list</h1>
+		<h1>todo list</h1>
 	</header>
 	<main>
-		<input id="userInput" type="text">
-		<button id="addButton">+</button>
+		<form action="">
+			<input placeholder="write down your goals :)" class="userInput" id="userInput">
+			<button class="addButton" id="addButton">+</button>
```

Comment:
É redundante teres uma `class` e um `id` com o mesmo valor
```suggestion
			<button id="addButton">+</button>
```

(O mesmo para o input de cima)